StatusValidator.py

```diff
@@ -5,3 +5,67 @@
 class StatusValidator:
     def __init__(self):
         pass
+
+    @classmethod
+    def isWin(cls, board: Board, player: Player, column: int) -> bool:
+        horizontalWin = StatusValidator.horizontalWin(board, player)
+        verticalWin = StatusValidator.verticalWin(board, player, column)
+        diagonalWin = StatusValidator.diagonalWin(board, player)
+
+        return horizontalWin or verticalWin or diagonalWin
+
+    def horizontalWin(board: Board, player: Player) -> bool:
```

Comment:
All methods in a class should have `self` as first parameter. (Unless they're `staticmethods` or `classmethods`.)